### PR TITLE
Switch to x/crypto/pkcs12 instead of Azure repo

### PR DESCRIPTION
**Pull request by paulmey**

### CODE REVIEW

**Comment by ahmetalpbalkan** [comment]
Can you vendor with godep?

**Comment by paulmey** [comment]
yup, one sec

**Comment by paulmey** [comment]
done

**Comment by ahmetalpbalkan** [comment]
Doesn't seem right. Did you use godep tool to vendor the pkg or just edited the json file? Source files are missing.

**Comment by paulmey** [comment]
I used the tool, but forgot to `git add` the godeps workspace

**Comment by phinze** [comment]
:+1: Just confirmed that this PR fixes the Terraform master build, which was broken when https://github.com/Azure/go-pkcs12/pull/26 landed.

File: management/publishSettings.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"fmt"
 	"io/ioutil"
 
-	"github.com/Azure/go-pkcs12"
+	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12"
```
Comment (ahmetalpbalkan) [inline management/publishSettings.go]: This line is wrong. The import path should be hust golang.org/...

**Comment by ahmetalpbalkan** [comment]
LGTM. Feel free to merge after import path fix.

File: management/publishSettings.go (hunk 1 above)
Comment (paulmey) [inline management/publishSettings.go]: See https://github.com/Azure/azure-sdk-for-go/blob/master/arm/compute/client.go#L22 for example. This is what godeps does to your import paths. This is not wrong, IMHO.

Comment (ahmetalpbalkan) [inline management/publishSettings.go]: that's what happens when you do `godep save -r` which rewrites your paths IMHO. I don't think we need path rewriting here but let's leave it as is for consistency until somebody complains. :)

Comment (ahmetalpbalkan) [inline management/publishSettings.go]: also keep in mind that if some project imports this package + `golang.org/x/crypto/pkcs12` separately, then it'll end up two copies of it, possibly even duplicate types with the same name in the runtime.

**Pull request merged**